Wikipedia/Code/MWKSection.m

```diff
@@ -244,7 +244,7 @@ - (nullable NSString *)summary {
     }] componentsJoinedByString:@" "] wmf_summaryFromText];
 }
 
-static NSString *const WMFSectionDisambiguationTitlesXPathSelector = @"//div[@class='hatnote']//a/@href";
+static NSString *const WMFSectionDisambiguationTitlesXPathSelector = @"//div[contains(@class, 'hatnote')]//a/@href";
 
 - (nullable NSArray<NSURL *> *)disambiguationURLs {
```

Comment:
Note: thinking we should use a bit of standard JS to mine these disambiguation URLs (and the page issues strings) from the web view's *existing* DOM instead of reconstructing a DOM via Hpple/XPath here. Android has such JS functions and these should be moved to the page-library soon-ish.